addons/runtime_debug_tools/scripts/debug_runtime_interaction_2d.gd

```diff
@@ -9,6 +9,9 @@ signal on_object_picked(node)
 
 var _is_active = false
 var _previous_camera : Camera2D
+var _initialMouseMode : Input.MouseMode
```

Comment:
Request - rename this `_previous_mouse_mode`.

- The naming convention here is `_underscore_case`.
- A similar pattern is being followed for restoring cameras, where the naming is `_previous_camera`.